src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.NativeAot.cs

```diff
@@ -1334,147 +49,13 @@ public static void RegisterForMarshalling(ComWrappers instance)
         /// <param name="fpQueryInterface">Function pointer to QueryInterface.</param>
         /// <param name="fpAddRef">Function pointer to AddRef.</param>
         /// <param name="fpRelease">Function pointer to Release.</param>
-        public static unsafe void GetIUnknownImpl(out IntPtr fpQueryInterface, out IntPtr fpAddRef, out IntPtr fpRelease)
+        public static unsafe partial void GetIUnknownImpl(out IntPtr fpQueryInterface, out IntPtr fpAddRef, out IntPtr fpRelease)
```

Comment:
```suggestion
        public static unsafe void GetIUnknownImpl(out IntPtr fpQueryInterface, out IntPtr fpAddRef, out IntPtr fpRelease)
```

Reply:
I currently have the implementations set up with NativeAOT and CoreCLR providing implementations of `partial` methods. That's why this is partial (the definition is in `ComWrappers.Common.cs` with implementations in CoreCLR and NativeAOT). I feel that this makes it a little clearer what methods a runtime must provide (and can optionally provide).

Let me know if you want me to not do that.

Reply:
My preference is style consistency. I believe that the typical style used for CoreLib is:

- The shared file is called Foo.cs (ie no special suffix like Common or Shared)
- The runtime or OS specific slice has special suffix, like Foo.CoreCLR.cs or Foo.Windows.cs
- There are no partial prototypes in the shared file for methods that are expected to be implemented in the runtime or OS specific file.